examples/barotropicqg_decaying_topography.jl

```diff
@@ -0,0 +1,233 @@
+#md # Also, it can be viewed as a Jupyter notebook via [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/barotropicqg_decaying_topography.ipynb).
+# 
+# An example of decaying barotropic quasi-geostrophic turbulence over topography.
+
+using FourierFlows, Plots, Printf, Random
+
+using Statistics: mean
+using FFTW: irfft
+
+import GeophysicalFlows.BarotropicQG
+import GeophysicalFlows.BarotropicQG: energy, enstrophy
+
+
+# ## Choosing a device: CPU or GPU
+
+dev = CPU()     # Device (CPU/GPU)
+nothing # hide
+
+
+# ## Numerical parameters and time-stepping parameters
+
+      n = 128            # 2D resolution = n²
+stepper = "FilteredRK4"  # timestepper
+     dt = 0.05           # timestep
+ nsteps = 2000           # total number of time-steps
+ nsubs  = 10             # number of time-steps for intermediate logging/plotting (nsteps must be multiple of nsubs)
+nothing # hide
+
+
+# ## Physical parameters
+
+L = 2π        # domain size
+nothing # hide
+
+# Define the topographic potential vorticity, ``\eta = f_0 h(x, y)/H``
+σx, σy = 0.4, 0.8
+topographicPV(x, y) = 3exp(-(x-1)^2/(2σx^2) -(y-1)^2/(2σy^2)) - 2exp(-(x+1)^2/(2σx^2) -(y+1)^2/(2σy^2))
+nothing # hide
+
+# ## Problem setup
+# We initialize a `Problem` by providing a set of keyword arguments. Not providing
+# a viscosity coefficient `ν` leads to the module's default value: `ν=0`. In this
+# example numerical instability due to accumulation of enstrophy in high wavenumbers
+# is taken care with the `FilteredTimestepper` we picked. The topophic PV is prescribed
+# via keyword argument `eta`.
+prob = BarotropicQG.Problem(dev; nx=n, Lx=L, eta=topographicPV, dt=dt, stepper=stepper)
+nothing # hide
+
+# and define some shortcuts
+sol, clock, vars, params, grid = prob.sol, prob.clock, prob.vars, prob.params, prob.grid
+x, y = grid.x, grid.y
+nothing # hide
+
+# and let's plot the topographic PV:
+contourf(grid.x, grid.y, params.eta',
+          aspectratio = 1,
+            linewidth = 0,
+               levels = 10,
+                    c = :balance,
+                 clim = (-3, 3),
+                xlims = (-grid.Lx/2, grid.Lx/2),
+                ylims = (-grid.Ly/2, grid.Ly/2),
+               xticks = -3:3,
+               yticks = -3:3,
+               xlabel = "x",
+               ylabel = "y",
+                title = "topographic PV η=f₀h/H")
+
+
+# ## Setting initial conditions
+
+# Our initial condition consist of a flow that has power only at wavenumbers with
+# ``6 < \frac{L}{2\pi} \sqrt{k_x^2 + k_y^2} < 12`` and initial energy ``E_0``:
+
+E₀ = 0.04 # energy of initial condition
+
+K = @. sqrt(grid.Krsq)                          # a 2D array with the total wavenumber
+
+Random.seed!(1234)
+qih = randn(Complex{eltype(grid)}, size(sol))
+@. qih = ifelse(K < 6, 0, qih)
+@. qih = ifelse(K > 12, 0, qih)
```

Comment:
Would it be informative to add an L/2*pi factor for the forcing bounds. I know it is 1 here, but it would be consistent with your forcing description above and could help people when modifying this code.

Reply:
Oops, that was not supposed to go away. Thanks!